blockly-core/core/ui/block_svg/block_svg.js

```diff
@@ -547,6 +547,11 @@ Blockly.BlockSvg.prototype.removeSpotlight = function() {
 Blockly.BlockSvg.prototype.render = function() {
   this.block_.rendered = true;
```

Comment:
Could this lead to an issue down the line where a component assuming the block was rendered when its SVG had not been?